pysite/database.py

```diff
@@ -11,7 +11,7 @@ class RethinkDB:
 
     def __init__(self, loop_type: str = "gevent"):
         self.host = os.environ.get("RETHINKDB_HOST", "127.0.0.1")
-        self.port = os.environ.get("RETHINKDB_PORT", "28016")
+        self.port = os.environ.get("RETHINKDB_PORT", "28015")
```

Comment:
28016 actually was there for a reason. it's because when we test the flask app locally, if we run rethinkdb with standard settings, it will spin up its admin panel on 8080 which is also where flask wants to run. so, as a solution, I just did `rethinkdb --port-offset 1`, which makes it run on 8081 and also 28016.

If you know a better way to do this that's fine too.

Reply:
the only reason I made this change is because the default port of rethinkdb is 28015, thought it easier to dev locally when using default settings in rethinkdb